cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaClusterState.java

```diff
@@ -244,13 +276,21 @@ public void writeOutputStream(OutputStream out, boolean verbose) throws IOExcept
     SortedSet<PartitionInfo> underReplicatedPartitions = new TreeSet<>(comparator);
     SortedSet<PartitionInfo> offlinePartitions = new TreeSet<>(comparator);
     SortedSet<PartitionInfo> otherPartitions = new TreeSet<>(comparator);
+    SortedSet<PartitionInfo> partitionsWithOfflineReplicas = new TreeSet<>(comparator);
 
-    populateKafkaPartitionState(underReplicatedPartitions, offlinePartitions, otherPartitions, verbose);
+    populateKafkaPartitionState(underReplicatedPartitions,
+                                offlinePartitions,
+                                otherPartitions,
+                                partitionsWithOfflineReplicas,
+                                verbose);
 
     // Write the cluster state.
     out.write(String.format("Offline Partitions:%n").getBytes(StandardCharsets.UTF_8));
     writeKafkaClusterState(out, offlinePartitions, topicNameLength);
 
+    out.write(String.format("Partitions with Offline Replicas:%n").getBytes(StandardCharsets.UTF_8));
+    writeKafkaClusterState(out, partitionsWithOfflineReplicas, topicNameLength);
```

Comment:
I think you need to also update writeKafkaClusterState method to print the offline replica information.